core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/install-packages/Debian.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install PostgreSQL packages ({{ ansible_os_family }})
+  apt:
+    name:
+      - postgresql-10
+      - postgresql-contrib-10
+      - python-psycopg2 # required for postresql ansible management
+    update_cache: yes
+    state: present
+
+- name: Install pgAudit package
+  apt:
+    name:
+      - postgresql-10-pgaudit
+    state: present
+  when:
+    - specification.extensions.pgaudit.enabled is defined
```

Comment:
What if pgaudit not defined?

Reply:
Then condition will be `false` and task will be skipped.